exp/lighthorizon/main.go

```diff
@@ -26,10 +27,17 @@ func main() {
 if left empty, uses a temporary directory`)
 	cacheSize := flag.Int("ledger-cache-size", defaultCacheSize,
 		"number of ledgers to store in the cache")
+	logLevelParam := flag.String("log-level", "info",
+		"logging level, info, debug, warn, error, panic, fatal, trace, default is info")
 	flag.Parse()
 
 	L := log.WithField("service", "horizon-lite")
-	L.SetLevel(log.InfoLevel)
+	logLevel, err := logrus.ParseLevel(*logLevelParam)
+	if err != nil {
+		logLevel = log.InfoLevel
```

Comment:
nit: Maybe we could spit out a warning here? e.g. 
```suggestion
		logLevel = log.InfoLevel
		log.Warnf("Failed to parse -log-level '%s', defaulting to 'info'.", *logLevelParam)
```